rosidl_generator_cpp/resource/msg__traits.hpp.em

```diff
@@ -60,11 +61,13 @@ for member in message.structure.members:
 @[end if]@
 @#>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
 @
-namespace rosidl_generator_traits
+@[for ns in message.structure.namespaced_type.namespaces]@
+namespace @(ns)
 {
 
```

Comment:
do we usually leave a blank line between nested namespaces?

i.e.

```cpp
namespace foo
{

namespace bar
{
```

vs

```cpp
namespace foo
{
namespace bar
{
```

I think this is generating code following the first style, though we typically use the second.

Feel free to ignore this if linters are happy.

Reply:
I copied this verbatim from `msg__struct.hpp.em`:

https://github.com/ros2/rosidl/blob/d1122a2d2cb0c2d048aad6453badc6bbd17800b5/rosidl_generator_cpp/resource/msg__struct.hpp.em#L83-L87